wmf/limitation_dsc.md

```diff
@@ -128,14 +128,14 @@ Start-DscConfiguration -UseExisting -CimSession $session
 
 IPv6 Addresses as Node Names in DSC configurations
 --------------------------------------------------
-IPv6 Addresses as node names in DSC configuration scripts is not supported in this release.
+IPv6 addresses as node names in DSC configuration scripts are not supported in this release.
```

Comment:
I think this might be better written as: "Using IPv6 addresses as node names in DSC configuration scripts is not supported in this reelase"